app/assets/javascripts/edit_ideas.js

```diff
@@ -0,0 +1,96 @@
+
+
+var inLineEditTitle = function(requestService){
+  $('#ideas').on('click', '.idea-title', function(){
+    var idea = this.closest('.idea')
+    var element = $(idea).find('.idea-title')
+
+    element.attr('contentEditable', true)
+
+    $(idea).on('keydown', function(e){
+      if (e.keyCode == 13){
+        e.preventDefault()
```

Comment:
I don't know if there is a default action that we _need_ to prevent in this case.